<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="mainTextField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonBold">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>26</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+B</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonItalic">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>26</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>I</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+I</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonUnderline">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>26</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>U</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+U</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>26</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ğŸ’¾</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+S</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonOpen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>26</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ğŸ“</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+O</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
